pkg/build/bazel/non_bazel.go

```diff
@@ -21,6 +21,16 @@ func BuiltWithBazel() bool {
 	return false
 }
 
+// Return true iff called from a test run by Bazel.
+func TestByBazel() bool {
```

Comment:
Let's not start function names with `Test` unless they're actually tests. I think since the signatures don't have a `*testing.T` argument, Go won't try to actually test these functions. But it is confusing and it obfuscates the results if anyone tries to find tests by grepping for `^func Test` or similar. Try `IsBazelTest()` or something like that.